lib/ansible/modules/packaging/language/pip.py

```diff
@@ -503,12 +571,24 @@ def main():
         # Automatically apply -e option to extra_args when source is a VCS url. VCS
         # includes those beginning with svn+, git+, hg+ or bzr+
         has_vcs = False
+
         if name:
             for pkg in name:
                 if bool(pkg and re.match(r'(svn|git|hg|bzr)\+', pkg)):
                     has_vcs = True
                     break
 
+            distributions = _get_distributions_from_names(module, name)
+            # check invalid combination of arguments
+            if version is not None:
+                if len(distributions) > 1:
+                    module.fail_json(msg="'version' is not available for installing multi-packages.")
+                if distributions[0].has_version_specifier:
+                    module.fail_json(msg="'version' argument conflicts with version specifier provided along with a package name.\
```

Comment:
You don't need to escape EOL. Just have two strings and Python will automatically know that it's in fact one string:

this
```python
(
"series "
"of"
" words"
)
```
equals
```python
"series of words"
```

Reply:
(but in case of function arg, you don't need additional braces)

Reply:
```python
func("one "
        "string",
        "another" " string")
```